### PR TITLE
set_all_attrs when init Field

### PR DESCRIPTION
**Pull request by ghost**
Description of the issue/feature this PR addresses:

Current behavior before PR: attributes from slots of the metaclass are first set when set_all_attr() is called in setup_attrs(). In the current branch there are built-in methods and addons which try to get access to attributes before setup_attrs() is called. That causes recursion depth exceeded problems in nearly every attribute and causes a huge performace problem in Odoo.

Desired behavior after PR is merged: now the attributes are created from slots - with an empty dict - when Field is initialized. Built-in methods and other classes can now access attributes before _setup_attrs() is currently called, and it will not cause recursion depth exceeded. It also makes Odoo a lot faster.

We could not see any side-effects at the moment. It is simply initialising the attributes to an empty dict. It does not change setup_attrs() or any Field which inherits from the base class.

At the moment it is only to prevent the recursion depth exceeded on attributes issue, but I recommend a review of the complete initialization process of class Field.

I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

Makes Odoo a lot faster because if the removal of the 'recursion depth exeeded' issue on attributes before set_all_attrs

### CODE REVIEW

File: addons/web/views/webclient_templates.xml [hunk 1]
```diff
@@ -100,7 +100,7 @@
             <link rel="stylesheet" href="/web/static/src/css/data_export.css"/>
             <link rel="stylesheet" href="/base/static/src/css/modules.css"/>
 
-            <link rel="stylesheet" href='/website/static/src/less/import_bootstrap.less'/>
+            <link rel="stylesheet" href='/web/static/src/less/import_bootstrap.less'/>
```
Comment (yelizariev) [inline addons/web/views/webclient_templates.xml]: @kkinateder you have to get rid of any updates non-related to ``set_all_attrs`` issue to get a non-zero chance be merged. You have to make a separate pull request for other updates.

Comment (kkinateder) [inline addons/web/views/webclient_templates.xml]: Have tryed, no idea, why every push to my branch is added to the PR. But as I can say yelizariev, there is more broken, in class Field, which is a Datadescriptor. Iam still in analysing, but as I can say until now, it seems, that the __getter__, __setter__ methods are not correctly, cause there is already __get__, __set__, __delete__ defined. So that recursion depth exceeded seems to be typical error coder does in a Datadescriptor. Watching https://www.youtube.com/watch?v=v2WTVCyTYMw  start at 50:10 .

Comment (yelizariev) [inline addons/web/views/webclient_templates.xml]: You need to create separate git branch and add there only one commit, that fix this issue only.
